Ideas/list of ideas.md

```diff
@@ -4,7 +4,8 @@
 An Artificial Neural Network or a Deep Learning Model that identifies whether the customer will leave the bank or not.
 ### 2. Parkinson Disease Classifier
 A machine learning model using multiple classification algorithms for analysis and prediction of Parkinson disease.
-
+### 3.Android App market on Google Play
```

Comment:
Can you please find the latest data source and link it here or maybe add it in the repo?

Reply:
Link for Data Source :
https://www.kaggle.com/lava18/google-play-store-apps
1).apps.csv : contains all the details of the applications on Google Play. There are 13 features that describe a given app.
2). user_reviews.csv: contains 100 reviews for each app, most helpful first. The text in each review has been pre-processed and attributed with three new features: Sentiment (Positive, Negative or Neutral), Sentiment Polarity and Sentiment Subjectivity.